examples/angular/src/app/resolver/dotcms-page.resolver.ts

```diff
@@ -11,44 +11,30 @@ import { DOTCMS_CLIENT_TOKEN } from '../client-token/dotcms-client';
  * @return {*}
  */
 export const DotCMSPageResolver = async (
-  route: ActivatedRouteSnapshot,
+  route: ActivatedRouteSnapshot
 ): Promise<{
   pageAsset: DotCMSPageAsset;
   nav: DotcmsNavigationItem;
 }> => {
   const client = inject(DOTCMS_CLIENT_TOKEN);
-
-  const url = route.url.map((segment) => segment.path).join('/');
   const queryParams = route.queryParams;
 
-  const pageProps = {
-    path: url || 'index',
-    language_id: queryParams['language_id'],
-    mode: queryParams['mode'],
-    variantName: queryParams['variantName'],
-    'com.dotmarketing.persona.id':
-      queryParams['com.dotmarketing.persona.id'] || '',
-  };
+  const pageAsset = route.data?.['pageAsset'] as DotCMSPageAsset;
 
   const navProps = {
     path: '/',
     depth: 2,
     languageId: queryParams['language_id'],
   };
 
-  const pageRequest = client.page.get(pageProps) as Promise<{
-    entity: DotCMSPageAsset;
-  }>;
   const navRequest = client.nav.get(navProps) as Promise<{
     entity: DotcmsNavigationItem;
   }>;
 
-  const [pageResponse, navResponse] = await Promise.all([
-    pageRequest,
+  const [navResponse] = await Promise.all([
```

Comment:
@rjvelazco I suggest if just there is one promise we could remove the `Promise.all`